cloud/jenkins/pxc_operator_eks_latest.groovy

```diff
@@ -321,12 +357,12 @@ pipeline {
             description: 'percona-xtradb-cluster-operator repository',
             name: 'GIT_REPO')
         string(
-            defaultValue: '1.24',
+            defaultValue: 'latest',
             description: 'EKS kubernetes version',
             name: 'PLATFORM_VER')
         choice(
             choices: 'NO\nYES',
```

Comment:
Let's set the default for CLUSTER_WIDE in all latest jobs to `YES` since we run it by default in our PRs with cluster wide.

Reply:
done